packages/linter/generators.ts

```diff
@@ -0,0 +1,4 @@
+export { lintInitGenerator } from './src/generators/init/init';
```

Comment:
Why is this file created?

Reply:
to avoid getting typescript not found error. however, i removed it.